doc/changes/DM-44668.misc.rst

```diff
@@ -0,0 +1 @@
+Update retryUnlessExit to allow multiple exit codes and add to documentation.  Add bps defaults for numberOfRetries, memoryMultiplier, and retryUnlessExit.
```

Comment:
Can we enclose the name of the parameters mentioned here in backquotes? So they visually stand out when this text is rendered to HTML.